main/2025/CG-01-28.md

```diff
@@ -14,6 +14,9 @@ No registration is required for VC meetings. The meeting is open to CG members o
 
 1. Opening
 1. Proposals and discussions
+   1. rounding variants proposal
+        1. IA refresher [Kloud Koder, 10 min]
+        2. rounding variants status report and phase 2 poll [whirlicote, 11 min]
```

Comment:
11 minutes is an oddly specific number. I expect that since it's been a while there will be some questions and discussion too; if this 10+11 minutes is for the presentation, maybe we should round up to 30 for discussion?